<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorFilePage</class>
 <widget class="QWidget" name="EditorFilePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>1193</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure file handling settings&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line2">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Open &amp;&amp; Close</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QCheckBox" name="clearBreakpointsCheckBox">
          <property name="toolTip">
           <string>Select, whether breakpoint belonging to an editor should be cleared, when the editor is closed</string>
          </property>
          <property name="text">
           <string>Clear Breakpoints upon closing</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="automaticReopenCheckBox">
          <property name="toolTip">
           <string>Select to reread the file automatically, if it was changed externally</string>
          </property>
          <property name="text">
           <string>Reread automatically when changed externally</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Warn, if file is greater than</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="warnFilesizeSpinBox">
          <property name="toolTip">
           <string>Enter the filesize, a warning dialog should be shown.</string>
          </property>
          <property name="suffix">
           <string> KB</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>16384</number>
          </property>
          <property name="singleStep">
           <number>16</number>
          </property>
          <property name="value">
           <number>1024</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>End of Line</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_8">
        <property name="title">
         <string>End of Line Characters</string>
        </property>
        <layout class="QHBoxLayout" name="_12">
         <item>
          <widget class="QRadioButton" name="lfRadioButton">
           <property name="toolTip">
            <string>Select Unix type end of line</string>
           </property>
           <property name="text">
            <string>Unix</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="crRadioButton">
           <property name="toolTip">
            <string>Select Macintosh type end of line</string>
           </property>
           <property name="text">
            <string>Macintosh</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="crlfRadioButton">
           <property name="toolTip">
            <string>Select Windows type end of line</string>
           </property>
           <property name="text">
            <string>Windows/DOS</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="automaticEolConversionCheckBox">
        <property name="toolTip">
         <string>Select whether the eol type should be converted upon opening the file.</string>
        </property>
        <property name="text">
         <string>Automatic End of Line Conversion</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Save</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="stripWhitespaceCheckBox">
        <property name="toolTip">
         <string>Select, whether trailing whitespace should be removed upon save</string>
        </property>
        <property name="text">
         <string>Strip trailing whitespace upon save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="createBackupFileCheckBox">
        <property name="toolTip">
         <string>Select, whether a backup file shall be generated upon save</string>
        </property>
        <property name="text">
         <string>Create backup file upon save</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="TextLabel13_3_3">
          <property name="text">
           <string>Autosave interval:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="autosaveSlider">
          <property name="toolTip">
           <string>Move to set the autosave interval in minutes (0 to disable)</string>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="autosaveLCD">
          <property name="toolTip">
           <string>Displays the selected autosave interval.</string>
          </property>
          <property name="digitCount">
           <number>2</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
          <property name="value" stdset="0">
           <double>5.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Encoding</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="advEncodingCheckBox">
        <property name="toolTip">
         <string>Select to use the advanced encoding detection </string>
        </property>
        <property name="whatsThis">
         <string>&lt;b&gt;Advanced encoding detection&lt;/b&gt;
&lt;p&gt;Select to use the advanced encoding detection based on the &amp;quot;universal character encoding detector&amp;quot; from &lt;a href=&quot;http://chardet.feedparser.org&quot;&gt;http://chardet.feedparser.org&lt;/a&gt;.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>Use advanced encoding detection</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Default Encoding:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="defaultEncodingComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Select the string encoding to be used.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Default File Filters</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Open Files:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="openFilesFilterComboBox"/>
      </item>
      <item row="0" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Save Files:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="saveFilesFilterComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Additional File Filters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&lt;b&gt;Note:&lt;/b&gt; Save file filters must contain one wildcard pattern only.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="openFiltersButton">
          <property name="toolTip">
           <string>Select to edit the open file filters</string>
          </property>
          <property name="text">
           <string>Open Files</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="savFiltersButton">
          <property name="toolTip">
           <string>Select to edit the save file filters</string>
          </property>
          <property name="text">
           <string>Save Files</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" rowspan="4">
       <widget class="QListWidget" name="fileFiltersList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="addFileFilterButton">
        <property name="text">
         <string>Add...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="editFileFilterButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Edit...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="deleteFileFilterButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="title">
      <string>File Preview</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Extensions of HTML files:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="previewHtmlExtensionsEdit">
        <property name="toolTip">
         <string>Enter the filename extensions of HTML files that may be previewed (separated by a space)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Extensions of Markdown files:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="previewMarkdownExtensionsEdit">
        <property name="toolTip">
         <string>Enter the filename extensions of Markdown files that may be previewed (separated by a space)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Extensions of ReST files:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="previewRestExtensionsEdit">
        <property name="toolTip">
         <string>Enter the filename extensions of ReStructuredText files that may be previewed (separated by a space)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Extensions of QSS files:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="previewQssExtensionsEdit">
        <property name="toolTip">
         <string>Enter the filename extensions of Qt Stylesheet files that may be previewed (separated by a space)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="previewRestSphinxCheckBox">
        <property name="toolTip">
         <string>Select to use 'sphinx' to generate the ReST preview</string>
        </property>
        <property name="text">
         <string>Use 'sphinx' for ReST preview</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>435</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>clearBreakpointsCheckBox</tabstop>
  <tabstop>automaticReopenCheckBox</tabstop>
  <tabstop>warnFilesizeSpinBox</tabstop>
  <tabstop>lfRadioButton</tabstop>
  <tabstop>crRadioButton</tabstop>
  <tabstop>crlfRadioButton</tabstop>
  <tabstop>automaticEolConversionCheckBox</tabstop>
  <tabstop>createBackupFileCheckBox</tabstop>
  <tabstop>stripWhitespaceCheckBox</tabstop>
  <tabstop>autosaveSlider</tabstop>
  <tabstop>advEncodingCheckBox</tabstop>
  <tabstop>defaultEncodingComboBox</tabstop>
  <tabstop>openFilesFilterComboBox</tabstop>
  <tabstop>saveFilesFilterComboBox</tabstop>
  <tabstop>openFiltersButton</tabstop>
  <tabstop>savFiltersButton</tabstop>
  <tabstop>fileFiltersList</tabstop>
  <tabstop>addFileFilterButton</tabstop>
  <tabstop>editFileFilterButton</tabstop>
  <tabstop>deleteFileFilterButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>autosaveSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>autosaveLCD</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
